### DISCUSSION TITLE
Just wondering....Surround music?

### DISCUSSION THREAD

**Discussion by kubycsolutions** [automatic-ripping-machine/automatic-ripping-machine#1148] [Q&A]
Should ARM do the Right Thing with surround-sound SACDs? Or does that require a decoding/encoding plugin?

(I presume that DVD-Audio surround sound works without complications.)

I've got just a few surround audio recordings, but it'd be nice to not have to do special handling for them.

**Comment by microtechno9000** [comment]
Until now had never heard of them. ARM uses the .abcde linux application to read, transcode and name music cd's, so if .abcde can process then ARM can, a quick search and it looks like they are formatted as a Direct Stream Transfer [Wiki](https://en.m.wikipedia.org/wiki/Super_Audio_CD). If you find a way that .abcde can read them let me know and I will see if ARM needs modification to read.

**Comment by JollyRgrs** [comment]
As far as I know, the way to rip SACD is with an SACD player. I purchased one for cheap on ebay, then you get some software on a USB stick, reboot the player and boom, you can rip over the network (or to USB storage I think). For DVD-A, I resorted to a software called DVD Audio Extractor (not free though) which works really good. for SACD, sometimes you have multiple options (stereo or multi-channel). For DVD-A, you have the high resolution audio (the DVD-A part) that may be stereo and/or multi but also the "DVD" playable portion that might be DTS or just DD or something else "DVD Compatible". I'm not so sure abcde would be a great option even if it can. Also, SACD can be a DSF file, but you may or may not want to convert it to FLAC depending on what you are playing it on (Plex handles DSF just fine, so does fubar2000).
